src/main/groovy/org/gradle/android/workarounds/room/KspWorkaround.groovy

```diff
@@ -22,9 +22,11 @@ class KspWorkaround extends AnnotationProcessorWorkaround<KspRoomSchemaLocationA
 
     @Override
     void initWorkaround() {
-        project.tasks.withType(kspTaskClass).configureEach {
-            if (kspIsAppliedWithRoom()) {
-                configureWorkaroundTask(it)
+        project.afterEvaluate {
```

Comment:
hi @runningcode, I know I know... 
But I just added the `project.afterEvaluate` to cover the case where `ksp` is only included in the project. We can guarantee that `KspJvmTask` exists once the project is evaluated. I sent you the video explaining the build classpath issue.

With this approach I can test directly in `nowinadroid` with the following results for experiment 3:
Before:
<img width="862" alt="Screenshot 2023-02-07 at 4 24 21 PM" src="https://user-images.githubusercontent.com/475733/217703084-7dd56710-53bc-4d55-a466-073254c9c580.png">

After:
<img width="1232" alt="Screenshot 2023-02-08 at 6 36 01 PM" src="https://user-images.githubusercontent.com/475733/217703227-237e367f-d78b-4143-b892-a0afb3592c6d.png">

(https://ge.solutions-team.gradle.com/c/6jamlbdawd2l4/odexxc7t7qteo/task-inputs?cacheability=cacheable)